test/integration/targets/openssh_cert/tasks/main.yml

```diff
@@ -239,6 +239,35 @@
           - "clear"
       valid_from: "2001-01-21"
       valid_to: "2019-01-21"
+  - name: Generate cert without serial
+    openssh_cert:
+      type: user
+      signing_key: '{{ output_dir }}/id_key'
+      public_key: '{{ output_dir }}/id_key.pub'
+      path: '{{ output_dir }}/id_cert_no_serial'
+      valid_from: always
+      valid_to: forever
+    register: rc_no_serial_number
+  - name: check default serial
+    assert:
+      that:
+        - "'Serial: 0' in rc_no_serial_number.info"
```

Comment:
There shouldn't be a default serial number.

Reply:
This default value is set by ssh-keygen:
```
$ man ssh-keygen | grep -B 1 -e 'default serial number'
     -z serial_number
             Specifies a serial number to be embedded in the certificate to distinguish this certificate from others from the same CA.  The default serial number is zero.
```

Reply:
Ah, sorry, didn't check.